js/fieldmanager.js

```diff
@@ -114,6 +114,25 @@ var match_value = function( values, match_string ) {
 	return false;
 }
 
+/**
+ * Initializes triggers to conditionally hide or show fields.
+ */
+var init_display_if = function() {
+	$( '.display-if' ).each(function() {
```

Comment:
I agree, although because this is existing FM code just relocated, I'd be inclined to determine a better scope in a separate PR. If that sounds OK to you, I'll file an issue.